bin/isolated-functions.sh

```diff
@@ -434,14 +434,21 @@ RC_INDENTATION=''
 RC_DEFAULT_INDENT=2
 RC_DOT_PATTERN=''
 
-case "${NOCOLOR:-false}" in
+
+
+if [[ -z "${NO_COLOR}" ]] ; then
+	case "${NOCOLOR:-false}" in
```

Comment:
I think the logic is correct now.

Very minor point, the quotes are not necessary in `[[ ]]` and `case`:
```suggestion
if [[ -z ${NO_COLOR} ]] ; then
	case ${NOCOLOR:-false} in
```